cmd/manager/main.go

```diff
@@ -40,13 +40,20 @@ func main() {
 		os.Exit(1)
 	}
 
-	// get watch namespace from environment variable
+	// Get watch namespace from environment variable
 	namespace, nsSpecified := os.LookupEnv("WATCH_NAMESPACE")
 	if !nsSpecified {
 		os.Exit(1)
 	}
 
-	log.Info(fmt.Sprintf("Watching namespace: %s", namespace))
+	// If namespace is a wildcard use the empty string to represent all namespaces
+	nsp := ""
```

Comment:
I'm generally in favour of longer variable names (unless it's some widely known abbreviation, such as cfg, mgr, etc), so maybe `namespaceWatched`? Bot a big deal though

Reply:
I renamed this to watchNamespace.